tests/test_core.py

```diff
@@ -62,6 +62,14 @@ def test_update_no_existing_destination(self):
         self.assertEqual(core.update(crate, lambda x: True)[0], Crate.CREATED)
         self.assertEqual(arcpy.Exists(crate.destination), True)
 
+    def test_update_no_deleted_destination(self):
```

Comment:
i should be able to understand what this test should do by the name. Can you be more verbose with what this test is testing so I don't have to read the code?